src/client/pages/OfferNew/Introduction/Sidebar/index.tsx

```diff
@@ -241,51 +237,32 @@ export const Sidebar: React.FC<Props> = ({
                 </Button>
 
                 <FooterExtraActions>
-                  {redeemedCampaigns.length === 0 && (
+                  {!hasRedeemedCampaigns && (
```

Comment:
I personally like the more explicit check, since here I have to think about whether JS treats "0" as a falsy/truthy value :P

Reply:
`hasRedeemedCampaigns` is a boolean in this case I believe

Reply:
Oh sorry, got confused by the diff